charts/monitoring-config/dashboards/app-requests.json

```
@@ -438,7 +441,7 @@
             "uid": "prometheus"
           },
           "editorMode": "code",
-          "expr": "avg(http_request_duration_seconds{namespace=\"${namespace}\", quantile=~\"${quantile}\"})",
+          "expr": "sum (increase(http_request_duration_seconds_sum{namespace=\"${namespace}\"}[$__rate_interval])) / sum (increase(http_request_duration_seconds_count{namespace=\"${namespace}\"}[$__rate_interval]))",
```

as above (only duplicating for convenience in case you wanna use the suggested changes feature)
```suggestion
          "expr": "sum (rate(http_request_duration_seconds_sum{namespace=\"${namespace}\"}[$__rate_interval])) / sum (rate(http_request_duration_seconds_count{namespace=\"${namespace}\"}[$__rate_interval]))",
```